fastlane/Fastfile

```diff
@@ -234,6 +234,43 @@ lane :run_pod_install do |options|
   end
 end
 
+desc "Run automatic bump in other repos"
+lane :bump_hybrid_dependencies do |options|
+  bump = current_version_number
+  repo_name = options[:repo_name],
+
+  UI.message("Triggering pipeline to bump #{repo_name} dependency to #{bump}")
+
+  require 'net/http'
+  require 'uri'
+  require 'json'
+
+  uri = URI.parse("https://circleci.com/api/v2/project/github/RevenueCat/purchases-flutter/pipeline")
+  request = Net::HTTP::Post.new(uri)
+  request.basic_auth(ENV['CIRCLECI_TOKEN'], "")
+  request.content_type = "application/json"
+  request.body = JSON.dump({
+    "branch" => "main",
+    "parameters" => {
+      "action" => "upgrade-hybrid-common",
+      "hybrid-common-version" => "#{bump}",
+    }
+  })
+
+  req_options = {
+    use_ssl: uri.scheme == "https",
+  }
+
+  response = Net::HTTP.start(uri.hostname, uri.port, req_options) do |http|
+    http.request(request)
+  end
+
+  puts response.code
+  puts response.body
+
+  UI.error("Error triggering the pipeline") unless response.code == "201"
```

Comment:
It would be great to also print what error happened, but that might be included if we print the code and body above.

Reply:
I think it will be helpful to print the error code in the message and remove the response.code log